src/components/Masthead/masthead.css

```
@@ -143,6 +143,14 @@ a.trusted-websites-link:hover {
   color: light-dark(#0151a0, #96c7f7);
 }
 
+a.trusted-websites-link:focus,
+a.trusted-websites-link:focus-visible,
+.sgds-masthead-button:focus,
+.sgds-masthead-button:focus-visible {
+  outline: 0;
+	box-shadow: 0px 0px 0px 4px #60aaf4;
```

We can use variable for this colour code a
```suggestion
	box-shadow: 0px 0px 0px 4px var(--sgds-blue-400)
```